lib/jxl_cms.cmake

```diff
@@ -24,13 +24,14 @@ target_include_directories(jxl_cms PUBLIC
   "$<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}>")
 
 set(JXL_CMS_PK_LIBS "")
+set(JPEGXL_CMS_LIBRARY_REQUIRES "")
 
 if (JPEGXL_ENABLE_SKCMS)
   target_link_libraries(jxl_cms PRIVATE skcms)
 else()
   target_link_libraries(jxl_cms PRIVATE lcms2)
   if (JPEGXL_FORCE_SYSTEM_LCMS2)
-    set(JXL_CMS_PK_LIBS "-llcms2")
+    set(JPEGXL_CMS_LIBRARY_REQUIRES "lcms2")
```

Comment:
I guess `JXL_CMS_PK_LIBS` could now be removed from here and the .pc.in template...

Reply:
I didn't remove it, since before this commit, JPEGXL_CMS_LIBRARY_REQUIRES was just sitting there unused.
So I figured they're kept around as easy to use future options.

Can easily remove it though, if preferred.

Reply:
yeah, let's remove it.